### PR TITLE
A tiny change to README.md#Installing

### PR DESCRIPTION
**Pull request by dvlprs**
## Description

Made it obvious one has to `cd` into their CakePHP directory and run the composer command from it.
## Summarize

Edited `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ Transparently use [Bootstrap 3][twbs3] with [CakePHP 3][cakephp].
 - Sample layouts (cover, signin, dashboard)
 - Bake templates *incomplete*
 
-## Installing
+## Installing Using [Composer][composer]
 
-Using [Composer][composer]:
+`cd` to your CakePHP directory and run the following command:
```
Comment (ADmad) [inline README.md]: Yeah "your app" or "your project" directory would be more appropriate.

**Comment by dereuromark** [comment]
I am not a too big fan of this, as it should be very clear when working with composer where to be for it to run (where the composer.json file is, so in the app dir).
This seems to be redundant and clear from the scope.

But meh, if it helps some people..

**Comment by dvlprs** [comment]
> But meh, if it helps some people..

My thoughts exactly. There's a chance someone will stop for a second and be uncertain about that. Not everyone has as much experience with composer as you do for instance. I'm unsure what exactly is wrong with saying "*your CakePHP directory*", but if you feel it's not okay then so be it. Rephrased that portion.

**Comment by dereuromark** [comment]
CakePHP dir = core dir (vendor/cakephp/cakephp) = dangerous tip.
My remark wasn't about rephrasing, I was just not sure this additional note is necessary.

**Comment by dvlprs** [comment]
> CakePHP dir = core dir (vendor/cakephp/cakephp) = dangerous tip.

I see what you're saying. Sorry about that then.

**Pull request merged**